.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "submodules/client-libs-java-samples"]
```

Comment:
don't we need to change this line as well ?:)

Reply:
This part of the .gitmodules file contains the name of the 'name' of the submodule, 
and it does not have to match the path to it.

To move the submodule I've used an intended git command:
`git mv submodules/ sample/` that has not changed the name. 

Nevertheless, even if it`s not required, it`s good to keep it in sync,
so I've just updated the name also. Thanks for spotting that!